packages/localize-tools/package.json

```diff
@@ -19,13 +19,14 @@
   "main": "lit-localize.js",
   "type": "module",
   "scripts": {
-    "clean": "git clean -dXn | sed 's/^Would remove //' | grep -v node_modules | xargs rm -rf",
+    "clean": "rimraf lib/",
```

Comment:
Oops yep you're right here, I was trying to figure out what the output would have been but just misunderstood it.

I believe this would be the correct command `"rimraf {fnv1a64,id-generation,lit-localize}.{js,js.map,d.ts} tests/ .tsbuildinfo examples/runtime/lib/ examples/transform/{bundled/,lib/}"`

Though the original is quite nice, I'll continue to look to see if I can find a similar alternative.

Hmm couple of thoughts though:
- does `git clean -dXn` always output `Would remove ` or are the responses translated to the users language?
- is there any risk of git clean somehow giving an error or unexpected result and end up `rm -rf` files you didn't want to like `*` or something?

Reply:
Updated the rimraf version for now.

Was hoping to be able to run the git clean directly like `git clean -dXf --exclude node_modules` but sadly can't get it to work as expected.

Could possibly have a mix of the two e.g.: 
`rimraf tests/ examples/runtime/lib/ examples/transform/{bundled/,lib/} && git clean -Xf`

Reply:
I'm fine with just enumerating the files for now.

> * does `git clean -dXn` always output `Would remove ` or are the responses translated to the users language?

Great question, I don't know. It's also possible for the string to change in English across versions, so it's probably pretty brittle anyway.

> * is there any risk of git clean somehow giving an error or unexpected result and end up `rm -rf` files you didn't want to like `*` or something?

You would have to have added the file you didn't actually want to delete to the `.gitignore`, so I think this would be pretty rare.

Reply:
Actually I just noticed that since https://github.com/lit/lit/pull/1831 has landed, and those symlinks are removed, there's no output for this package in the root, so the simple `rimraf lib/` should work fine.